inbm/dispatcher-agent/dispatcher/sota/snapshot.py

```diff
@@ -223,10 +227,13 @@ def recover(
         dispatcher_state.clear_dispatcher_state()
         if self.snap_num:
             self._rollback_and_delete_snap()
+            logger.debug("Rebooting to recover from failed SOTA...")
```

Comment:
Consider removing the redundant log message before the conditional reboot. The log message inside the conditional block is sufficient to indicate a reboot due to a failed SOTA. [important]

Reply:
It may not reboot, so we want to leave that debug statement.